lib/collection/src/shards/transfer/transfer_tasks_pool.rs

```diff
@@ -60,6 +86,10 @@ impl TransferTaskProgress {
         self.points_total = total;
         self.eta.set_progress(transferred);
     }
+
+    pub fn total(&self) -> usize {
+        max(self.points_total, self.points_transferred)
+    }
```

Comment:
Nit: we don't need the new method if we put cap on the field itself:

```suggestion
        self.points_total = max(total, transferred);
        self.eta.set_progress(transferred);
    }
```